cltk/lemmatize/latin/unsupervised.py

```diff
@@ -0,0 +1,382 @@
+'''Scan the entire Tesserae corpus of Latin and build frequency data
+for each possible lemma in the corpus. This results in a simple, unsupervised language model
+of frequency data. Use the unsupervised language model created by lemma_frequency.py in order
+to lemmatize Latin words. Picking the most common lemma, as in this module tests out at > 90%
+accurate. Contributors: James Gawley, Jeff Kinnison.
+'''
+
+import os
+from os import listdir
+from os.path import isfile, join, expanduser
+import pickle
+import string
+from operator import itemgetter
+from cltk.tokenize.word import WordTokenizer
+from cltk.stem.latin.j_v import JVReplacer
+from cltk.semantics.latin.lookup import Lemmata
+
+class FrequencyModel:
+    '''Generate unsupervised count of lemma frequencies in the Tesserae Latin text corpus.'''
+
+    def __init__(self):
+        self.lemmatizer = Lemmata(dictionary='lemmata', language='latin')
+        self.jv = JVReplacer()
+        self.word_tokenizer = WordTokenizer('latin')
+        self.count_dictionary = dict()
+        self.punctuation_list = ['!', ';', ':', '?', '-', '–', '&', '*',
+                                 '(', ')', '[', ']', ',', '"', '\'']
```

Comment:
If one could pass in an override to the init function, one might be able to use this FrequencyModel class on other corpora.
Same for the choice of language for the Lemmata, and WordTokenizer.

I know, this is placed in the Latin package module directory, but it looks like something we might want to generify and make available as more common paradigm. And after all, the class is named FrequencyModel and not LatinFrequencyModel ;-)